src/main.c

```diff
@@ -1,7 +1,7 @@
 /*
  *   Software Updater - client side
  *
- *      Copyright © 2012-2016 Intel Corporation.
```

Comment:
Why this change? Why only to 2016 if you are updating the `from`?

Reply:
You are reviewing "files changed" and GitHub gets confused about the file renaming.

What is really happening is that we first moved contents of main.c to update.c, and then renamed swupd.c to main.c.

Reply:
Ah weird. ok I see it correctly when I view the commits.